docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md

```diff
@@ -15,48 +15,241 @@ related:
+      {% endinfo_block %}
 
-        The next step varies based on the selected calculator type:
+   2. Select the **DISCOUNT APPLICATION TYPE** and define which products are eligible for the discount.
+   3. To proceed to the **Conditions** tab, click **Next**.
+4. On the **Conditions** tab, do the following:
+    1. Select the **APPLY WHEN** conditions or click **Plain query** and enter the query manually.
+    2. Enter the value for **THE DISCOUNT CAN BE APPLIED IF THE QUERY APPLIES FOR AT LEAST X ITEM(S).** field.
+5. To create the new discount, click **Save**.
 
-          a. *Fixed amount*: Enter the amounts (per currency and price mode, if applicable) to be discounted.
+### Reference information: Creating cart rule discounts
 
-          b. *Percentage*: Enter the percent value to be discounted.
+This section contains reference information you select and enter when creating cart rule discounts.
 
-        {% endinfo_block %}
+#### General information tab
 
-    2. Select the **DISCOUNT APPLICATION TYPE** and define which products are eligible for the discount. For reference information, in the **Create new discount** reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
- * Click **Next**, or select the **Conditions** tab to proceed.
- * On the **Conditions** tab, do the following:
-    1. Select the **APPLY WHEN** conditions or click **Plain query** and enter the query manually. For reference information, see [Conditions](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#conditions).
-    2. Enter the value for **THE DISCOUNT CAN BE APPLIED IF THE QUERY APPLIES FOR AT LEAST X ITEM(S).** field.
-* To create the new discount, click **Save**.
+The following table describes the attributes you enter and select on the **General information** tab:
+
+<div class="width-100">
+
+| ATTRIBUTE |DESCRIPTION  |
+| --- | --- |
+| STORE RELATION | Stores that your cart rule will be active in. You can select multiple stores.|
+| DISCOUNT TYPE | Drop-down list where you select either **Voucher codes** or **Cart rule** discount type. |
+| NAME | Unique name that will be displayed in the cart, in the **Available discounts** section on the Storefront, if the cart rule is valid. Should be short, but descriptive. |
+| DESCRIPTION | Explains the voucher and helps a customer understand why they are eligible for the discount and what they can receive. The description will be displayed in the **Cart calculation** section along with the applied discount amount if the discount is applicable to a product in the cart.|
+| PRIORITY | Defines [the discount priority](/docs/scos/user/features/{{page.version}}/promotions-discounts-feature-overview.html#discount-priority). Represented as an integer value from `1` to `9999`, `1` being the highest priority and `9999` the lowest. |
```

Comment:
Before providing a link, try to briefly explain what priority is.

Reply:
Done